src/System.Device.Gpio/System/Device/Gpio/Board.cs

```diff
@@ -0,0 +1,265 @@
+﻿using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+
+namespace System.Device.Gpio
+{
+    public class Board
+    {
+        /// <summary>
+        /// The Raspberry Pi model.
+        /// </summary>
+        public enum Model
+        {
+            /// <summary>
+            /// Unknown model.
+            /// </summary>
+            Unknown,
+
+            /// <summary>
+            /// Raspberry Model A.
+            /// </summary>
+            RaspberryPiA,
+
+            /// <summary>
+            /// Model A+.
+            /// </summary>
+            RaspberryPiAPlus,
+
+            /// <summary>
+            /// Model B rev1.
+            /// </summary>
+            RaspberryPiBRev1,
+
+            /// <summary>
+            /// Model B rev2.
+            /// </summary>
+            RaspberryPiBRev2,
+
+            /// <summary>
+            /// Model B+.
+            /// </summary>
+            RaspberryPiBPlus,
+
+            /// <summary>
+            /// Compute module.
+            /// </summary>
+            RaspberryPiComputeModule,
+
+            /// <summary>
+            /// Pi 2 Model B.
+            /// </summary>
+            RaspberryPiB2,
+
+            /// <summary>
+            /// Pi Zero.
+            /// </summary>
+            RaspberryPiZero,
+
+            /// <summary>
+            /// Pi Zero W.
+            /// </summary>
+            RaspberryPiZeroW,
+
+            /// <summary>
+            /// Pi 3 Model B.
+            /// </summary>
+            RaspberryPiB3,
+
+            /// <summary>
+            /// Pi 3 Model B+.
+            /// </summary>
+            RaspberryPiB3Plus,
+
+            /// <summary>
+            /// Compute module 3.
+            /// </summary>
+            RaspberryPiComputeModule3,
+        }
+        #region Fields
+
+        private static readonly Lazy<Board> board = new Lazy<Board>(LoadBoard);
```

Comment:
This seems to be unused

Reply:
You're right. I used it in my former code. must be deleted.